utility/bins8k.h

```diff
@@ -20,6 +20,7 @@ namespace HL {
   enum { NUM_BINS = 32 };
   enum { NUM_SIZECLASSES = 1018 };
 
+  // FIXME: is this page size dependent?
```

Comment:
This one is definitely based on the page size.

Reply:
I just noticed that the `BIG_OBJECT` definitions are somewhat inconsistent across bin sizes. Is this intentional, and which ones should be based on page size rather than hardcoded? (Note that the below definitions are from the master branch, *not* my PR.)

```
utility/bins64k.h
51:    enum { BIG_OBJECT = 8192 };

utility/bins8k.h
23:  enum { BIG_OBJECT = 4096 - sizeof(Header) };

utility/binspow2.h
67:    enum { BIG_OBJECT = MaxSize };

utility/bins4k.h
22:      enum { BIG_OBJECT = 4096 - sizeof(Header) };

utility/bins.h
73:    enum { BIG_OBJECT = Size / 8 }; // - sizeof(Header) };

utility/bins16k.h
26:  enum { BIG_OBJECT = 16384 - sizeof(Header) };

heaps/general/dlheap.h
248:  enum { BIG_OBJECT = 2147483648U };

```

Reply:
Oh, I also thought I should mention that `bins8k` does not seem to be used by Mesh since making the syntax invalid has no effect on Mesh's build. I'm assuming it's still used in other downstream libraries though.

Reply:
The change in size was intentional. Let's just leave them as is.